packages/cli-platform-apple/src/commands/runCommand/createRun.ts

```diff
@@ -310,15 +310,8 @@ const createRun =
     }
 
     if (args.udid) {
-      const device = devices.find((d) => d.udid === args.udid);
-      if (!device) {
-        return logger.error(
-          `Could not find a device with udid: "${chalk.bold(
-            args.udid,
-          )}". ${printFoundDevices(devices)}`,
```

Comment:
Cna we still print this?

Reply:
absolutely - thanks again for the review - I think I focused in too much on the macCatalyst case and while I was still emitting a similar warning below, it was without the devices list which I think is really helpful in the general case.

So I put that back and re-pushed. What do you think?

Reply:
@thymikee re-pushed, hopefully addressing your concern. Cheers 👋